apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.IntegrationTests/BigtableAssert.cs

```diff
@@ -0,0 +1,171 @@
+﻿// Copyright 2017 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Bigtable.V2.IntegrationTests
+{
+    internal static class BigtableAssert
+    {
+        public static async Task HasNoValuesAsync(
+            BigtableClient client,
+            TableName tableName,
+            BigtableByteString rowKey)
+        {
+            // TODO: Use cleaner API when available
+            var response = client.ReadRows(new ReadRowsRequest
+            {
+                TableName = tableName.ToString(),
+                Rows = new RowSet { RowKeys = { rowKey } }
+            });
+
+            while (await response.ResponseStream.MoveNext(default))
+            {
+                var current = response.ResponseStream.Current;
+                foreach (var chunk in current.Chunks)
+                {
+                    Assert.True(false, $"No chunks expected for the row at {rowKey.Value.ToStringUtf8()}");
+                }
+            }
+        }
+
+        public static async Task HasSingleValueAsync(
```

Comment:
The class is named `...Assert`, so I think `BigtableAssert.AssertSingleValueAsync` is a bit redundant. I was going with something you could read as Bigtable**Assert** that it **Has** a single value (asynchronously), almost like `Assert.DoesNotContain` - normally you'd expect `DoesNotContain` to return a bool, but not in the context of assert.

I think I'll keep it as-is rather than going with a `Get...` alternative for 2 reasons:
* It doesn't make sense in the concept of the `...Assert` class, so it means adding a new place for this common code (not the end of the world though).
* It would be verifying everything specified (family name, column name, version) but then it's the callers responsibility to verify the value. Seems easier for the callers to just have it do everything.